src/drivers/imu/analog_devices/adis16507/Analog_Devices_ADIS16507_registers.hpp

```diff
@@ -64,35 +64,28 @@ namespace Analog_Devices_ADIS16507
 {
 static constexpr uint32_t SPI_SPEED = 2 * 1000 * 1000;       // 2 MHz SPI serial interface
 static constexpr uint32_t SPI_SPEED_BURST = 1 * 1000 * 1000; // 1 MHz SPI serial interface for burst read
-
 static constexpr uint32_t SPI_STALL_PERIOD = 16; // 16 us Stall period between data
 
-static constexpr uint16_t DIR_WRITE = 0x80;
-
-static constexpr uint16_t Product_identification = 0x407B;
-
 static constexpr uint32_t SAMPLE_INTERVAL_US = 500; // 2000 Hz
 
-enum class Register : uint16_t {
-	DIAG_STAT      = 0x02,
-
-	FILT_CTRL      = 0x5C,
-
-	RANG_MDL       = 0x5E,
-
-	MSC_CTRL       = 0x60,
-
-	GLOB_CMD       = 0x68,
-
-	FIRM_REV       = 0x6C, // Identification, firmware revision
-	FIRM_DM        = 0x6E, // Identification, date code, day and month
-	FIRM_Y         = 0x70, // Identification, date code, year
-	PROD_ID        = 0x72, // Identification, part number
-	SERIAL_NUM     = 0x74, // Identification, serial number
+static constexpr uint16_t DIR_WRITE = 0x80;
 
-	FLSHCNT_LOW    = 0x7C, // Output, flash memory write cycle counter, lower word
-	FLSHCNT_HIGH   = 0x7E, // Output, flash memory write cycle counter, upper word
-};
+namespace Register
+{
+static constexpr uint16_t DIAG_STAT = 0x02;
```

Comment:
Why?

Reply:
Previously it was an enum type. I changed to namespace to keep the same syntax because it looks nice, but without the convolution of having to use the enum type `Register` everywhere. Personally I don't like creating strong types for standard types since it obfuscates the underlying type and its one more abstraction you have to carry around in your head when navigating the code